cpp/open3d/t/pipelines/slam/Model.h

```diff
@@ -75,6 +75,27 @@ class Model {
                               float trunc_voxel_multiplier = 8.0,
                               bool enable_color = true);
 
+    /// Track using PointToPlane depth odometry.
+    /// \param input_frame Input RGBD frame.
+    /// \param raycast_frame RGBD frame generated by raycasting.
+    /// \param depth_scale Scale factor to convert raw data into meter metric.
+    /// \param depth_max Depth truncation to discard points far away from the
+    /// camera
+    /// \param method Method used to apply RGBD odometry.
+    /// \param criteria Criteria used to define and terminate iterations.
+    /// In multiscale odometry the order is from coarse to fine. Inputting a
+    /// vector of iterations by default triggers the implicit conversion.
+
+    odometry::OdometryResult TrackFrameToModel(
+            const Frame& input_frame,
+            const Frame& raycast_frame,
+            float depth_scale,
+            float depth_max,
+            float depth_diff,
+            odometry::Method method,
```

Comment:
also overload method here to `odometry::Method::PointToPlane`

Reply:
we can remove the later `TrackFrameToModel`. i.e. we can just pass the default argument and remove function overloading.

Reply:
How worried are are you about version compatibility details? The only reason I didn't do it this way from the start is that it'll no longer have the original symbol in the shared object and could break applications if they try to do a drop in replacement shared library update. If that isn't a concern I agree this is the cleaner approach

Reply:
Both parameters have default values (after adding default for odometry_method), therefore existing applications based on this won't be affected.

Reply:
This is true only if the application is rebuilt against the this version or later. If an application has been built against the released v0.15 and wants to try dropping in a shared object from this commit onward, that application might crash due to the original function signature not being present in the newer shared object. 

This might not be a thing you care about, especially since it's still in the major version 0 beta stage, and if that is the case  I'll resolve these as you put them, but it is something I wanted to call out in case there was some desire to keep backwards compatibility on binaries.